app/components/view-popup.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+	Dialog,
+	DialogContent,
+	DialogFooter,
+	DialogHeader,
+	DialogTitle,
+	DialogTrigger,
+} from "@/components/ui/dialog";
+import type { DangerKey } from "@/lib/danger-levels";
+import type { Sensor } from "@/lib/sensors";
+import { cn } from "@/lib/utils";
+import { DialogDescription } from "@radix-ui/react-dialog";
+import { t } from "i18next";
+import { PopupNotes } from "./daily-notes";
+import { Button } from "./ui/button";
+import { Card } from "./ui/card";
+import type { Event } from "./weekly-view";
```

Comment:
This import should be this:

```suggestion
import type { WeekEvent } from "@/features/week-widget/types";
```
dont understand how it still works though ahahah